src/components/AbstractSelectableComponent.js

```diff
@@ -146,10 +160,25 @@ export default class AbstractSelectableComponent extends React.Component {
     return this.renderImages(unprojectedProps);
   }
 
+  onViewStateChange({ viewState }) { // eslint-disable-next-line class-methods-use-this
```

Comment:
You are using `this`, so `eslint-disable-next-line class-methods-use-this` shouldn't be necessary?

Reply:
Yes, good point, I have removed it